src/helpers/update-request.js

```diff
@@ -0,0 +1,12 @@
+const Request = require('@lulibrary/lag-alma-utils/src/request')
+
+// Helper method for SNS Handler Lambdas to write new Request data to the cache
+module.exports = (requestData, resourceData) => {
+  const requestID = requestData.user_request.request_id
+  const requestCacheTable = resourceData.requestTable.name
+  const eventRequest = new Request({ id: requestID, tableName: requestCacheTable, region: resourceData.requestTable.region })
```

Comment:
Is `eventRequest` a library item request?

Reply:
Yeah, it's the specific Alma request that triggered the webhook -> lambda -> sns chain of events